steps/members_steps.py

```diff
@@ -0,0 +1,78 @@
+from utils import request_util
+import json
+from compare import expect
+import collections
+
+@given(u'I send a POST request to "{endpoint}"')
+def step_impl(context, endpoint):
+
+    query_params = ({"key": context.key}, {"token": context.token})
+    for row in context.table:
+        query_params += ({row["key"]: row["value"]}, )
+
+    context.response = request_util.post_request(context.base_url + endpoint, query_params)
+
+    data = context.response.json()
+    idRecord = data['id']
+    context.boardId = idRecord
+
+@given(u'I add a new member to "{endpoint}"')
+def step_impl(context,endpoint):
+    query_params = ({"key": context.key}, {"token": context.token})
+
+    #Adding values from context.table to query_params to be sent on the requests.
+    for row in context.table:
+        query_params += ({row["key"]: row["value"]}, )
+
+    #Getting board id, baords_steps didn't updated because are of general use.
+    data = context.response.json()
+    idRecord = data['id']
+    query_params += ({"id": idRecord}, )
+
+    endpoint = endpoint.replace('{id}',idRecord)
+    context.response = request_util.put_request(context.base_url + endpoint, query_params)
+
+    #Save member Id to be used later
+    data = context.response.json()
+    idRecord = data['id']
+    context.recordId = idRecord
+
+@when(u'I send a GET request to "{endpoint}"')
+@then(u'I send a GET request to "{endpoint}"')
+def step_impl(context,endpoint):
+    query_params = ({"key": context.key}, {"token": context.token})
+
+    endpoint = endpoint.replace('{id}',context.boardId)
+    context.getResponse = request_util.get_request(context.base_url + endpoint, query_params)
+    context.response = context.getResponse
+
+@when(u'I delete a member using "{endpoint}"')
```

Comment:
you may use this implementation already defined:
@step(u'I send a {method} request to "{endpoint}"')